backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/compatibility/properties/EStringIfDescriptionProvider.java

```diff
@@ -39,7 +39,7 @@
 public class EStringIfDescriptionProvider {
     private static final String IF_DESCRIPTION_ID = "EString"; //$NON-NLS-1$
 
-    private static final String TEXTFIELD_DESCRIPTION_ID = "Textfield"; //$NON-NLS-1$
+    private static final String TEXTAREA_DESCRIPTION_ID = "Textarea"; //$NON-NLS-1$
```

Comment:
Why does this file needs to be changed? Using a textfield does not change anything at all on the behavior of the frontend nor on this feature, right?

Reply:
Ths is needed because the frontend component TextfieldPropertySection checks the typeName if it equals 'Textarea'